util/compression_test.cc

```diff
@@ -1768,119 +1769,266 @@ TEST_F(DBAutoSkip, AutoSkipCompressionManager) {
     ASSERT_OK(Flush());
   }
 }
-class CostAwareTestFlushBlockPolicy : public FlushBlockPolicy {
+class AutoTuneFlushBlockPolicy : public FlushBlockPolicy {
  public:
-  explicit CostAwareTestFlushBlockPolicy(const int window,
-                                         const BlockBuilder& data_block_builder)
-      : window_(window),
+  explicit AutoTuneFlushBlockPolicy(const int wait_block_count,
+                                    const BlockBuilder& data_block_builder)
+      : wait_block_count_(wait_block_count),
         num_keys_(0),
-        data_block_builder_(data_block_builder) {}
+        data_block_builder_(data_block_builder) {
+    def_cpu_prediction_ = 500;
```

Comment:
`def` is even harder to understand then `default` with trivial benefit of saving 4 characters.